git_machete/github.py

```diff
@@ -123,6 +123,8 @@ def fire_github_api_get_request(method: str, url: str, token: Optional[str], req
 
         if 200 <= response.status < 300:
             return parsed_response_body
```

Comment:
While you're here... `fire_github_api_get_request` should probably be made private